bigtable/CMakeLists.txt

```diff
@@ -27,7 +27,7 @@ set(DOXYGEN_PROJECT_NUMBER
     "${BIGTABLE_CLIENT_VERSION_MAJOR}.${BIGTABLE_CLIENT_VERSION_MINOR}.${BIGTABLE_CLIENT_VERSION_PATCH}")
 set(DOXYGEN_EXAMPLE_PATH examples)
 set(DOXYGEN_PREDEFINED "BIGTABLE_CLIENT_NS=v${BIGTABLE_CLIENT_VERSION_MAJOR}")
-set(DOXYGEN_EXCLUDE_PATTERNS "*/client/internal/*;*/client/testing/*;*/*_test.cc")
+set(DOXYGEN_EXCLUDE_PATTERNS "*/client/internal/*")
```

Comment:
Why did the exclude pattern change?

Reply:
I thought that tests can be included once collaboration graph is removed, I will revert the changes in exclude pattern and submit the PR.